webextensions/api/scripting.json

```diff
@@ -165,6 +165,102 @@
               "safari_ios": "mirror"
             }
           },
+          "InjectionResult": {
+            "__compat": {
+              "support": {
+                "chrome": {
+                  "version_added": "88"
+                },
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "102"
+                },
+                "firefox_android": "mirror",
+                "opera": "mirror",
+                "safari": {
+                  "version_added": "15.4"
+                },
+                "safari_ios": "mirror"
+              }
+            },
+            "documentId": {
```

Comment:
@Rob--W given the other discussions, should I remove this from here?

Reply:
Sure.